firmware/run_formatter.sh

```diff
@@ -0,0 +1,3 @@
+find . \( -name '*.c' -o -name '*.h' -o -path './dependencies' -prune \) -type f >> formatter_files.txt
+uncrustify -c firmware_style.cfg -F formatter_files.txt --no-backup
```

Comment:
This requires uncrustify to be installed on the system so I think we should run this script in the docker container